### DISCUSSION TITLE
Upload and assign to album in one step

### DISCUSSION THREAD

**Discussion by philippkanne** [immich-app/immich#11570] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I currently use the immich app like this:
- select images
- upload to server
- select images again and create an album (or add to an album)

I would be happy if I could select the album(s) directly when uploading. 

There is a similar request see [#10959](https://github.com/immich-app/immich/discussions/10959)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1678
